content/_help/verify-your-identity/verify-your-identity-in-person._es.md

```diff
@@ -0,0 +1,103 @@
+
+## Inicie este proceso en Login.gov antes de ir a la oficina de correos
+
+Tenga preparada la siguiente información para ingresarla en Login.gov:
+
+1. Su documento de identidad emitido por su estado. [¿No tiene un documento de identidad estatal?](/es/help/verify-your-identity/accepted-state-issued-identification/)
+2. Número de Seguridad Social.
+3. Un número de teléfono con un plan de telefonía que esté a su nombre.
+   * Si no tiene un plan de telefonía que esté a su nombre, podemos enviarle el código de verificación por correo, lo cual tarda aproximadamente de 3 a 5 días.
+
+Si no puede completar alguno de estos pasos, póngase en contacto con el centro de ayuda de la agencia asociada para obtener ayuda.
+
+## Lleve los siguientes artículos a la oficina de correos
+
+### Una copia del código de barras enviado a su correo electrónico
+
+Una vez que haya completado los pasos en Login.gov, le enviaremos por correo electrónico un código de barras y una fecha límite para ir a la oficina de correos. Puede imprimir el correo electrónico o mostrarlo desde su dispositivo móvil. Si va a la oficina de correos después de la fecha límite, su información no se guardará y tendrá que iniciar de nuevo el proceso.
+
+### Documento de identidad emitido por el estado
+
+Lleve el mismo documento de identidad que utilizó en Login.gov. Su identificación no debe estar vencida. En este momento, solo se aceptan las siguientes identificaciones emitidas por el estado: 
+
+* Licencia de conducir de los 50 estados, del Distrito de Columbia (DC) y de otros territorios de Estados Unidos (Guam, Islas Vírgenes de Estados Unidos, Islas Marianas y Puerto Rico)
+* Una identificación emitida por el estado que no sea una licencia de conducir
+  * Se trata de un documento de identidad emitido por el estado, el Distrito de Columbia (DC) o un territorio de EE.UU. que confirme la identidad, pero no otorga privilegios de conducción.
+
+Actualmente no aceptamos otras formas de identificación, como pasaportes o carnés militares. No podrá verificar su identidad en la oficina de correos sin un documento de identidad emitido por el estado. Actualmente estamos trabajando para añadir más formas de verificar su identidad.
+
+Hasta entonces, póngase en contacto con el centro de ayuda de la agencia asociada para saber qué puede hacer.
+
+### Comprobante de domicilio actual
+
+Si su dirección actual no coincide con la que figura en su documento de identidad, tendrá que llevar un comprobante de su dirección actual a la oficina de correos. Los comprobantes de domicilio aceptados son:
+
+* Contrato de arrendamiento, hipoteca o escritura de fideicomiso.
+* Registro de votación.
+* Tarjeta de registro del vehículo.
+* Póliza de seguro de hogar o de vehículo.
+
+Puede traer una copia física o mostrar una copia digital del documento. No puede mostrar una foto o una captura de pantalla del documento.
```

Comment:
Preparing suggestion in case we need to swap it out:



```suggestion
Debe traer una copia en físico del documento original. Está prohibido mostrar una foto o una captura de pantalla del documento.
```